include/SFML/Graphics/Font.hpp

```diff
@@ -373,6 +399,7 @@ class SFML_GRAPHICS_API Font
     void*                      m_streamRec;   //!< Pointer to the stream rec instance (it is typeless to avoid exposing implementation details)
     void*                      m_stroker;     //!< Pointer to the stroker (it is typeless to avoid exposing implementation details)
     int*                       m_refCount;    //!< Reference counter used by implicit sharing
+    bool                       m_isSmooth;    //!< Status of the smooth filter
     Info                       m_info;        //!< Information about the font
```

Comment:
- Missing default initialization in constructor
- Missing copy in copy constructor
- Missing copy in assignment operator

Reply:
Done